### PR TITLE
Ensures the Swift container list call requests JSON as per spec

### PR DESCRIPTION
**Pull request by zack-shoylev**

### CODE REVIEW

**Comment by zack-shoylev** [comment]
https://issues.apache.org/jira/browse/JCLOUDS-1080

File: apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/features/ContainerApi.java [hunk 1]
```diff
@@ -77,6 +78,7 @@
    @Named("container:list")
    @GET
    @Fallback(EmptyFluentIterableOnNotFoundOr404.class)
+   @QueryParams(keys = {"format"}, values = {"json"})
```
Comment (andrewgaul) [inline apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/features/ContainerApi.java]: Don't need an array for single values.

**Comment by andrewgaul** [comment]
:+1: Can we confirm that other clients also send this header?

**Comment by zack-shoylev** [comment]
Other clients?

**Comment by andrewgaul** [comment]
python-swiftclient or similar?

**Comment by zack-shoylev** [comment]
swift on ubuntu:

DEBUG:swiftclient:REQ: curl -i https://storage101.ord1.clouddrive.com/v1/MossoCloudFS_0d629063-aa0f-4bf9-9e04-62e1fd6aaa2c?format=json -X GET -H "X-Auth-Token:

Good to get confirmation. It's in the spec and other clients use it. Still having some issues with the path test though.

**Comment by zack-shoylev** [comment]
merged

**Pull request closed without merging**